storage/sql_migrations/001_initial.sql

```diff
@@ -0,0 +1,5 @@
+-- migrate:up
```

Comment:
I think we an remove this first migration, now we have more (and we're changing tools anyways)